.github/workflows/check-examples.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         python -m pip install wheel
         python -m pip install -r requirements.txt
         python -m pip install -c constraints.txt jupyter openfermion plotly
-        python -m pip install docker p2j pytest
+        python -m pip install docker p2j pytest black==22.6.0
```

Comment:
Maybe we should better add black with the specific version to the global requirements file?

Reply:
Yes, we could do this. I did not put it in the `requirements.txt` because if you use `pre-commit`, then the tool would automatically download the black version specified in the `.pre-commit-config.yaml`, and then run the checks.  But I guess having it in the requirements file is good for those that won't use `pre-commit`.